WordPress/WordPressTest/ItemProviderMediaExporterTests.swift

```diff
@@ -76,7 +76,9 @@ final class ItemProviderMediaExporterTests: XCTestCase {
 
     // MARK: - Video
 
-    func testThatVideoIsExported() throws {
+    // This test is too slow to run on the CI as it sometimes takes more than
+    // a couple of seconds. Uncomment it if you are making changes to the exporter.
+    func xtestThatVideoIsExported() throws {
```

Comment:
I decided to disable it for now because it's unacceptable for one test to take this much time. Maybe I could find a smaller video that would take less time to get transcoded. I used one of the existing files.